node/node.go

```diff
@@ -407,11 +407,11 @@ func (node *AlgorandFullNode) startMonitoringRoutines() {
 
 	// PKI TODO: Remove this with #2596
 	// Periodically check for new participation keys
-	go node.checkForParticipationKeys()
+	go node.checkForParticipationKeys(node.ctx.Done())
```

Comment:
@AlgoStephenAkiki fyi - there is going to be at least this merge conflict with your integration branch.